docs/operator-manual/index.md

```diff
@@ -1,6 +1,6 @@
 # Overview
 
-This guide is for administrator and operator wanting to install and configure Argo CD for other developers.
+This guide is for administrators and operators wanting to install and configure Argo CD for other developers.
 
 !!! note
-    Please make sure you've completed the [getting started guide](../getting_started.md).
\ No newline at end of file
+    Please make sure you've completed the [getting started guide](../getting_started.md).
```

Comment:
Where is the difference between these two lines?

Reply:
I used the github editor to make this change and it changed the newline (and I didn't notice). I will remove it.

Reply:
Ah, it's okay yo have it.